### PR TITLE
Migration/UI toolkit semantic tokens v 0.7.9 alpha2

### PR DESCRIPTION
**Pull request by harshpathakzz**
## What does this PR do?
Migrates primitive tokens to semantic tokens 

## What packages have been affected by this PR?
`ui-toolkit`

## Types of changes

What types of changes does your code introduce to this project?

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)

- [x] New feature (non-breaking change which adds functionality)

- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Package version increase in any of the packages?
0.7.9-alpha

## Checklist before merging

_Put an `x` in the boxes that apply_

- [ ] These changes have been thoroughly tested.

- [ ] Changes need to be immediately published on npm.

### CODE REVIEW

File: packages/ui-toolkit/src/components/atoms/Popover/Popover.tsx [hunk 1]
```diff
@@ -34,16 +34,16 @@ const Popover = (props: Props) => {
 
   return (
     <div
-      className="pop12Wrapper"
-      onClick={showTip}
-      onMouseEnter={showTip}
-      onMouseLeave={hideTip}
-      onTouchStart={hideTip}
-    >
+        className="pop12Wrapper"
```
Comment (shubhamgavali1998) [inline packages/ui-toolkit/src/components/atoms/Popover/Popover.tsx:37]: revert these spaces

**Review by AkshayNaikGroww — CHANGES_REQUESTED** [review]

File: packages/ui-toolkit/src/components/atoms/Popover/Popover.tsx [hunk 2]
```diff
@@ -43,7 +43,7 @@ const Popover = (props: Props) => {
       {children}
       {
         isPopoverVisible && (
-          <div className={`pop12Popover borderPrimary backgroundPrimary ${popoverWrapperClass} ${direction || POPOVER_POSITIONS.TOP}`}>
+          <div className={`pop12Popover contentPrimary borderPrimary borderPrimary backgroundPrimary ${popoverWrapperClass} ${direction || POPOVER_POSITIONS.TOP}`}>
```
Comment (AkshayNaikGroww) [inline packages/ui-toolkit/src/components/atoms/Popover/Popover.tsx:46]: borderPrimary applied twice

File: packages/ui-toolkit/src/components/atoms/Tabs/Tabs.tsx [hunk 3]
```diff
@@ -81,7 +80,7 @@ const getActiveTabDimensions = (data: Tab[], activeIndex: number) => {
 
   } else {
     if (typeof document !== 'undefined') {
-      const prevActiveElement = document?.getElementsByClassName('tabs8TextActive') as HTMLCollectionOf<HTMLElement>;
+      const prevActiveElement = document?.getElementsByClassName('tabs8TextActive contentAccent contentAccent') as HTMLCollectionOf<HTMLElement>;
```
[the remark this answers is not in the document]
Comment (AkshayNaikGroww) [reply packages/ui-toolkit/src/components/atoms/Tabs/Tabs.tsx:83]: still present

File: packages/ui-toolkit/src/components/atoms/Toastify/toastify.css [hunk 4]
```diff
@@ -29,15 +28,13 @@
...
 
 .sb10ToastifySuccess {
-  background-color: var(--green500);
 }
 
 .sb10ToastifyError {
-  background-color: var(--red500);
 }
 
 .sb10ToastifyInfo {
```
[the remark this answers is not in the document]
Comment (AkshayNaikGroww) [reply packages/ui-toolkit/src/components/atoms/Toastify/toastify.css:30]: Replace with util class

**Review by AkshayNaikGroww — CHANGES_REQUESTED** [review]

**Review by AkshayNaikGroww — APPROVED** [review]

**Pull request merged**